Automattic/jetpack
# Migration plugin: Add track events and fix font face

Closes https://github.com/Automattic/wp-calypso/issues/73135
Closes https://github.com/Automattic/dotcom-forge/issues/1681

## Proposed changes:
* Installed missing font face
* Added tracking events for:
  * jetpack_migration_get_started_click
  * jetpack_migration_check_progress_click

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a testing JN site
* Navigate to Move to WordPress.com
* Check if the heading font is Recoleta (see screenshot below)
* Click on `Get Started` and check if everything works as before since we attach the record event

## Screenshot
<table>
  <tr>
    <td><img width="611" alt="Screenshot 2023-02-15 at 16 05 50" src="https://user-images.githubusercontent.com/1241413/219069006-2514fe0c-4e2c-42bd-850d-2bca023c35fe.png">
    <td><img width="570" alt="Screenshot 2023-02-15 at 16 06 00" src="https://user-images.githubusercontent.com/1241413/219069068-0893d617-e1a5-4312-a2b3-0bb955d4e3cf.png">
